python/tank_vendor/shotgun_authentication/__init__.py

```diff
@@ -20,5 +20,9 @@
 from tank.authentication import ShotgunAuthenticationError, AuthenticationError, IncompleteCredentials, AuthenticationCancelled
 from tank.authentication import ShotgunUser, deserialize_user, serialize_user
 
-
-
+def get_logger():
```

Comment:
Apparently this was missing from our compatibility shim with older tk-core. It's been broken for a long while, which makes me think that nobody actually had this problem, but it did block QA from testing so it's worth fixing anyway.